.github/workflows/deltaengine.yaml

```
@@ -0,0 +1,160 @@
+  extract_version:
+    name: extract_version
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/heads/release-v') 
+    outputs:
+      version: ${{ steps.extract_version.outputs.version }}   
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      - name: Extract Version
+        id: extract_version
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)